apps/ledger-live-mobile/src/logic/usePromptBluetoothCallback.ts

```diff
@@ -1,70 +1,58 @@
 import { useCallback, useState } from "react";
-import { useTranslation } from "react-i18next";
-import { NativeModules, Platform, ToastAndroid } from "react-native";
+import { NativeModules, Platform } from "react-native";
 
 const { BluetoothHelperModule } = NativeModules;
 
-const {
-  E_ACTIVITY_DOES_NOT_EXIST,
-  E_BLE_CANCELLED,
-  E_ENABLE_BLE_UNKNOWN_RESPONSE,
-  E_SECURITY_EXCEPTION,
-  E_UNKNOWN_ERROR,
-} = BluetoothHelperModule;
+const { E_BLE_CANCELLED } = BluetoothHelperModule;
 
 type AndroidError = {
   code: string;
   message: string;
 };
 
+export enum BluetoothPromptResult {
+  OK,
+  UNAUTHORIZED,
+}
+
 /**
  * @returns a function that turns on bluetooth on the phone
  */
 export function usePromptBluetoothCallback() {
-  const { t } = useTranslation();
-  return useCallback(async (): Promise<boolean> => {
+  return useCallback(async (): Promise<number> => {
     try {
-      return await NativeModules.BluetoothHelperModule.prompt();
+      const result = await NativeModules.BluetoothHelperModule.prompt();
+      return parseInt(result, 10);
```

Comment:
Does `usePromptBluetoothCallback` returns a number or a `BluetoothPromptResult` number enum ? The question is: how do we know that `prompt` won't produce a value > 1 ?